src/Compilers/Core/Portable/InternalUtilities/OneOrMany.cs

```diff
@@ -89,7 +104,7 @@ public OneOrMany<T> Add(T one)
 
         public bool Contains(T item)
         {
-            if (HasOne)
+            if (HasOneItem)
```

Comment:
nit: could remove this 'if' check altogether. the 'foreach' logic below is efficient and correct for any representation of OneOrMany.